Yet on one machine the connection.on("connect" event is never called.